atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -566,7 +566,7 @@ def get_build(self, metadata):
                     extra['filesystem_koji_task_id'] = task_id
 
         # Append media_types from pulp pull
-        pulp_pull_results = self.workflow.exit_results.get(PLUGIN_PULP_PULL_KEY)
+        pulp_pull_results = self.workflow.postbuild_results.get(PLUGIN_PULP_PULL_KEY)
```

Comment:
We should attempt to grab either from postbuild_results or exist_results.

Reply:
koji_promote runs only in prod, and pulp_pull in prod runs only as postbuild

Reply:
ah right, ok